index.html

```diff
@@ -2557,7 +2573,8 @@
         <ol class="algorithm">
           <li>If the <var>request</var>.<a>[[\updating]]</a> is true, then
           terminate this algorithm and take no further action. Only one update
-          may take place at a time. This should never occur.
+          may take place at a time. The <a>user agent</a> user interface should
```

Comment:
Hi Marcos,

Just a note that my proposal does not introduce the word MAY. The proposed sentence is: "The <a>user agent</a> user interface should ensure that this never occurs." The replaced sentence was "This should never occur."

Here's why I propose the change:

* Cnsistency with 5 instances of that sentence elsewhere in the document. (There are 2
instances of the replaced sentence in the document, so I went with the majority formulation.)
* I think that formulating this as a user agent responsibility is more useful than stating existentially
that something should not occur.

Ian

P.S. I will update the pull request to take into account your other suggestions.